### DISCUSSION TITLE
Pipeline tasks execution and Workers (nodes)

### DISCUSSION THREAD

**Discussion by emirot** [concourse/concourse#8984] [General]
Hello,

I could not get from the documentation here https://concourse-ci.org/internals.html 
If all the tasks in the pipeline are assigned to the same node or not? Or Is it configurable ?
Use case, for expensive workload to execute some tasks in a pipeline  in parallel across different nodes so it scales horizontally.

Thanks

**Comment by suhlig** [comment]
It is determined by the [container placement strategy](https://concourse-ci.org/container-placement.html).
